apps/web/src/pages/ManagerDashboardPage/ManagerDashboardPage.tsx

```diff
@@ -0,0 +1,440 @@
+                                  children: "Opened on: April 30th, 2024",
+                                },
+                              ]}
+                              buttonName="IT-02: Database architect"
+                            />
+                          </PreviewList.Root>
+                        </div>
+                      </Accordion.Content>
+                    </Accordion.Item>
+                  </Accordion.Root>
+                </TaskCard.Item>
+              </TaskCard.Root>
+            </div>
+            {/* Sidebar */}
+            <div
+              data-h2-display="base(flex)"
+              data-h2-flex-direction="base(column)"
+              data-h2-gap="base(x1)"
+              data-h2-max-width="p-tablet(x14)"
+            >
+              {/* Switch to new component in #11031 */}
+              <ResourceBlock.Root
+                headingColor="quinary"
+                headingAs="h2"
+                title={intl.formatMessage({
+                  defaultMessage: "Current role",
+                  id: "C0LMCq",
+                  description: "Card title for a nav role switcher",
+                })}
+              >
+                <ResourceBlock.Item
+                  title="Manager"
+                  href="#"
+                  description={intl.formatMessage({
+                    defaultMessage:
+                      "Easily switch between roles your account has access to.",
+                    id: "gPvdHC",
+                    description: "Helper instructions for a nav role switcher",
+                  })}
+                ></ResourceBlock.Item>
+              </ResourceBlock.Root>
+              <ResourceBlock.Root
+                headingColor="quaternary"
+                headingAs="h2"
+                title={intl.formatMessage({
+                  defaultMessage: "Your information",
+                  id: "jALTj0",
+                  description: "Card title for an information card",
+                })}
+              >
+                {showUnfinishedPieces ? (
+                  // This block is missing an href since the page doesn't exist yet.  It also needs logic to dynamically set the state.
+                  <ResourceBlock.Item
+                    state="complete"
+                    title={intl.formatMessage({
+                      defaultMessage: "Manager profile",
+                      id: "hkvlOx",
+                      description: "Link to manager profile page",
+                    })}
+                    href="#"
+                    description={intl.formatMessage({
+                      defaultMessage:
+                        "Describe your leadership style and team culture to give applicants an idea of what working with you is like.",
+                      id: "lbtzDG",
+                      description:
+                        "Helper instructions for a 'manager profile' card",
+                    })}
+                  />
```

Comment:
Do we not have the logic setup for getting the profile state? I believe it's `isProfileComplete` that can be fetched in the `me` query.

Reply:
Discussion in standup: No, the manager profile is different from the applicant profile represented by `isProfileComplete`.